Makefile

```diff
@@ -294,3 +294,9 @@ lint:
 
 _lint:
 	pre-commit run --all-files
+
+test:
+	skipper make _test
```

Comment:
we need an option to run specific test, possible? lets just open ticket for that